integration-libs/cdc/components/login/facade/facade-providers.ts

```diff
@@ -0,0 +1,15 @@
+import { Provider } from '@angular/core';
+import { LoginFormComponentService } from 'feature-libs/user/account/components/login-form';
+import { CdcJsService } from 'integration-libs/cdc/root';
+import { AuthService } from 'projects/core/src/auth';
+import { GlobalMessageService } from 'projects/core/src/global-message';
+import { WindowRef } from 'projects/core/src/window';
+import { CdcLoginComponentService } from '.';
+
+export const facadeProviders: Provider[] = [
+  {
+    provide: LoginFormComponentService,
+    useClass: CdcLoginComponentService,
```

Comment:
For consistency, could you align the naming with the original `LoginFormComponentService`, please: 
`CdcLoginComponentService` -> `CdcLoginFormComponentService`

Reply:
agreed, would be following the naming of files as in the core library.
- `LoginFormComponentService` -> `CDCLoginFormComponentService`
- `RegisterComponentService`-> `CDCRegisterComponentService`